frontEnd/src/components/hackathonDetails/details.jsx

```diff
@@ -7,53 +7,60 @@ import HandleDay from '../hackathonLister/handleDay';
 import HandleTime from '../hackathonLister/handleTime';
 
 function Details() {
-    const { id } = useParams();
+    const { id  } = useParams();
     const [hackathon,setHackathon]=useState([])
 
+    console.log(hackathon.Themes?.[0]);
     const fetchData=(url)=>{
         fetch(url).then((res)=>res.json())
         .then((res)=>{
-            setHackathon(res)   
+            setHackathon(res)  
         }).catch((error)=>{
-            console.log(error)
+            console.log('Error occured')
         })
     }
 
     useEffect(()=>{
-        fetchData(`http://localhost:1003/api/hackathons/${id}`);
+        fetchData(`http://localhost:2003/api/hackathons/${id}`);
     },[])
     return (
         <div>
             <Navbar />
             <div className="flex flex-col px-10% relative top-14">
                 <div>
                     <img
-                        src={hackathon.image}
+                        src={hackathon.Poster}
                         alt=""
                         className="h-100 w-1220 rounded"
                     />
                 </div>
                 <div className="flex gap-2 items-center text-emperor font-bold text-sm mt-5">
                     <span>
-                        <HandleDate date={hackathon.date} format={'short'} />
+                        <HandleDate date={hackathon.Date} format={'short'} />
                     </span>
                     <img src={fullStop} className="h-4" />
-                    {hackathon.tags?.map((data) => <span key={data}>{data}</span>)}
-                     
+                    {hackathon.Themes && hackathon?.Themes[0].split(',').map((tag) => (
```

Comment:
Is the split by `,` required? I guess there is a logical error here which takes only the 1st entry from the `Themes` array

Reply:
The value of Themes is an array of themes like ['ai','ml']. To split that , I used split(',')

Reply:
If the value of `Themes` is an array then split wouldn't work as split works on a `String` object. Ref - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split